Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rasa/rasa-sdk:2.2.0
+
+COPY actions /app/actions
+
+USER root
+RUN pip install --no-cache-dir -r /app/actions/requirements-actions.txt
+
+USER 1001
+CMD ["start", "--actions", "actions", "--debug"]
```

Comment:
why do we need this, isn't docker image built by the github action?

Reply:
I added it back because it's needed to test that the docker image builds & runs locally e.g. to make sure all requirements are present.

Reply:
Ah okay, so that will go away before merge?

Reply:
No, it should stay there IMO so that users can build the image locally without writing their own Dockerfile.